spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/TestHelperBase.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iceberg.spark;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import org.apache.spark.sql.Row;
+import org.assertj.core.api.Assertions;
+import org.junit.Assert;
+
+public class TestHelperBase {
```

Comment:
I don't think we need this class, we can just use `SparkTestHelperBase` because there's nothing JUnit4-specific in there

Reply:
Yes, that's correct, I remove `TestHelperBase` and add changes to `SparkTestHelperBase` with JUnit5 updates.